src/Symfony/Bundle/FrameworkBundle/Tests/Routing/RouterTest.php

```diff
@@ -82,6 +82,32 @@ public function testGenerateWithServiceParamWithSfContainer()
         $this->assertSame('"bar" == "bar"', $router->getRouteCollection()->get('foo')->getCondition());
     }
 
+    public function testGenerateWithDefaultLocale(): void
```

Comment:
`void` should be removed, we don't use it on tests

Reply:
Removed.